python_orocos_kdl/mainpage.dox

```diff
@@ -1,26 +0,0 @@
-/**
```

Comment:
@MatthijsBurgh Just curious why this file is removed.

Reply:
Documentation is generated by Sphinx, so any files related to Doxygen should be removed.

Reply:
Makes sense.